crates/bundler/src/bundler.rs

```diff
@@ -55,10 +58,19 @@ impl Bundler {
             .clone()
             .get_transaction_count(self.wallet.signer.address(), None)
             .await?;
+        let balance = client
+            .clone()
+            .get_balance(self.wallet.signer.address(), None)
+            .await?;
+
         let mut tx: TypedTransaction = ep
             .handle_ops(
                 uos.clone().into_iter().map(Into::into).collect(),
-                self.beneficiary,
+                if balance < self.min_balance {
+                    self.wallet.signer.address()
+                } else {
+                    self.beneficiary
+                },
```

Comment:
Maybe putting some logs here so that people know when and who is being charged and funding.

Reply:
I am gonna add some logging, makes sense. Charged is being always the account derived from seed phrase that is passed in CLI, but beneficiary can be any address.